src/components/variable.jsx

```diff
@@ -107,7 +109,7 @@ const Variable = React.createClass({
                 {startDate && stopDate &&
                   <h3>Du <FormattedDate value={startDate} /> au <FormattedDate value={stopDate} />&nbsp;:</h3>
                 }
-                <pre><code>{formulas[date].content}</code></pre>
+                <code><SyntaxHighlighter language='python' style={tomorrow}>{formulas[date].content}</SyntaxHighlighter></code>
```

Comment:
@Anna-Livia @fpagnoux I have tried `react-syntax-highlighter` on subsets of the code. The result isn't a text block/square but a highlight of every line or substring. And the syntax analyse doesn't seem to work (one colour is applied).
`react-highlight` accepts jsx code. I'm working on the highlighting itself.